packages/core-plugin-api/api-report.md

```diff
@@ -644,6 +644,17 @@ export type PendingOAuthRequest = {
   trigger(): Promise<void>;
 };
 
+// @public
+export type PinnipedSupervisorApi = {
+  getClusterScopedIdToken(
+    audience: string,
+    options?: AuthRequestOptions,
+  ): Promise<string>;
+};
+
+// @public
+export const pinnipedSupervisorAuthApiRef: ApiRef<PinnipedSupervisorApi>;
```

Comment:
👋 , I'd like to have a thinking about whether this is the best place to provide this, seeing as it's a pretty specific interface. One option to consider is whether this should simply be a k8s frontend module or smth like that instead.

Ideally it would implement the `ProfileInfoApi` and `SessionApi` as well btw, so that it can be listed in the settings page.

Reply:
Hi Patrik 👋 , I placed Pinniped AuthApiRef on `core-plugin-api` just as a convenience since others `AuthApiRefs` are there. As you said, it is a little bit different as other AuthApiRefs so perhaps it should reside on a different location, so I’m open to move it to a `kubernetes-react` or a new pinniped front end plugin, let me know what you prefer. 

We don’t expect to use Pinniped as an authentication provider to sign-in and identify users, only as a way to provide access to k8s resources on multiple k8s clusters, so at some point, we could have a user authenticated against multiple k8s clusters, each cluster having their respective certs/token. Let me know if you think we should implement those interfaces, I would need to investigate a little bit more since we would need to receive the `audience` parameter, but open to do it.

Reply:
Patrik, did you have a chance to think about if `kubernetes-react` is a good place  ?

Reply:
Hello Patrick, I tried to move the code to `kubernetes-react` but I got this error.

`'@backstage/plugin-kubernetes-react' with role 'web-library' has a runtime dependency on package '@backstage/core-app-api', which is not permitted. If you are using this dependency for dev server purposes, you can move it to devDependencies instead.`

I would appreciate any guidance that you could give me.

I had to export some classes and types on `core-app-api` too, so it could be easy to move the code to a different package.